File: API/API/Models/Account.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace API.Models
{
    public class Account
    {
        public int Id { get; set; }
        [Required]
        public int AccountNumber { get; set; }
        [Required]
        public int Balance { get; set; }
        [Required]
        public string Firstname { get; set; }
        [Required]
        public string Lastname { get; set; }
        [Required]
        public int Age { get; set; }
        [Required]
        public string Gender { get; set; }
        [Required]
        public string Address { get; set; }
        [Required]
        public string Employer { get; set; }
        [Required]
        [DataType(DataType.EmailAddress)]
        [EmailAddress]
        public string Email { get; set; }
        [Required]
        public string City { get; set; }
        [Required]
        public string State { get; set; }
    }
}

File: Portal1/src/app/models/account.ts

export class AccountModel{
  id: number;
  accountNumber: number;
  balance: number;
  firstname: string;
  lastname: string;
  age: number;
  gender: string;
  address: string;
  employer: string;
  email: string;
  city: string;
  state: string;
}
export enum Accounts
{
  AccountNumber,
  Balance,
  Firstname,
  Lastname,
  Age,
  Address,
  Employer,
  Email,
  City,
  State,
}

